.vscode/settings.json

```diff
@@ -7,5 +7,6 @@
         "candle-pyo3"
     ],
     "python.testing.unittestEnabled": false,
-    "python.testing.pytestEnabled": true
+    "python.testing.pytestEnabled": true,
+    "rust-analyzer.cargo.features": ["metal"]
```

Comment:
Wouldn't that break the editor for non metal users?

Reply:
I'm still seeing the line that enables metal. Maybe I'm missing something?

Reply:
Fixed, sorry!